.github/workflows/linux-build-base.yml

```diff
@@ -109,10 +109,10 @@ jobs:
           else
             # cuDF (unsupported for Clang) and Faiss (link issue when using Clang)
             # are excluded for Clang compilation and need to be added back when using GCC.
-            EXTRA_CMAKE_FLAGS+="-DVELOX_ENABLE_CUDF=ON"
-            EXTRA_CMAKE_FLAGS+="-DVELOX_ENABLE_FAISS=ON"
+            EXTRA_CMAKE_FLAGS+=("-DVELOX_ENABLE_CUDF=ON")
+            EXTRA_CMAKE_FLAGS+=("-DVELOX_ENABLE_FAISS=ON")
             # Investigate issues with remote function service: Issue #13897
-            EXTRA_CMAKE_FLAGS+="-DVELOX_ENABLE_REMOTE_FUNCTIONS=ON"
+            EXTRA_CMAKE_FLAGS+=("-DVELOX_ENABLE_REMOTE_FUNCTIONS=ON")
```

Comment:
It seems that these changes are already merged in main by https://github.com/facebookincubator/velox/pull/14552 .

But the merge caused a CI failure: https://github.com/facebookincubator/velox/issues/14869

The CI failure can be fixed by the change for `velox/experimental/cudf/tests/utils/ParquetConnectorTestBase.cpp` in this PR or #14870.